src/Handler.php

```diff
@@ -163,7 +163,7 @@ public function exceptionHandler($throwable)
     public function errorHandler($errno, $errstr, $errfile = '', $errline = 0)
     {
         if (!$this->client->shouldIgnoreErrorCode($errno)) {
-            $report = Report::fromPHPError($this->client->getConfig(), $errno, $errstr, $errfile, $errline);
+            $report = Report::fromPHPError($this->client->getConfig(), $errno, $errstr, $errfile, $errline, false, Report::EXCEPTION_HANDLER);
```

Comment:
This severity reason should probably be ERROR_CLASS because we set the severity based on the error raised